src/WingetCreateCore/Common/PackageParser.cs

```diff
@@ -643,6 +643,37 @@ private static bool ParseExeInstallerType(string path, Installer baseInstaller,
             }
         }
 
+        /// <summary>
+        /// Checks if a MSI Installer database was generated by WiX, based on common characteristics.
+        /// </summary>
+        /// <param name="installer">A MSI Installer database.</param>
+        /// <returns>A boolean.</returns>
+        private static bool IsWix(QDatabase installer)
+        {
+            // Do any of the WiX properties exist?
+            foreach (var property in installer.Properties)
+            {
+                if (property.Property.ToLower().Contains("wix"))
+                {
+                    return true;
+                }
+            }
```

Comment:
```suggestion
            if Array.Exists( installer.Properties, property => property.Property.ToLower().Contains("wix")) return true;
```

Reply:
Note that you could remove the `if` and `return true` here by or-ing them with your other two statements